### ISSUE TITLE
lost entire post

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#825]
```
What's the problem?

Tried to publish a post. Said it published, but it didn't. Went to reopen 
scribefire and the composition was completely gone.  This happened twice.



What browser are you using?


Chrome.


What version of ScribeFire are you running?
Don't know--I uninstalled it. Too risky.


```

-----
Original issue reported on code.google.com by `jamy.ba...@gmail.com` on 19 Mar 2014 at 8:34

**Comment by GoogleCodeExporter** [comment]
```
what CMS or Blog Type are you using?
```

-----
Original comment by `scribefi...@gmail.com` on 7 Apr 2014 at 8:55

**Comment by GoogleCodeExporter** [comment]
```
Blogger
```

-----
Original comment by `jamy.ba...@gmail.com` on 7 Apr 2014 at 11:53

**Comment by GoogleCodeExporter** [comment]
```
we found the issue for this and will include it with our next release
```

-----
Original comment by `scribefi...@gmail.com` on 8 Apr 2014 at 3:06

**Comment by GoogleCodeExporter** [comment]
```
Fixed for blogger blogspot in version 4.3.1
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 6:58
* Changed state: **Fixed**
